README.md

```diff
@@ -173,14 +174,15 @@ wheels for `torch`, `torchvision`, and `torch_xla` at
 
 | Package | Cloud TPU VMs Wheel (XRT on Pod, Legacy Only) |
 | --- | ----------- |
-| torch_xla | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch_xla-2.0-cp38-cp38-linux_x86_64.whl` |
-| torch | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch-2.0-cp38-cp38-linux_x86_64.whl` |
-| torchvision | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torchvision-2.0-cp38-cp38-linux_x86_64.whl` |
+| torch_xla | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch_xla-2.1-cp38-cp38-linux_x86_64.whl` |
+| torch | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch-2.1-cp38-cp38-linux_x86_64.whl` |
+| torchvision | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torchvision-2.1-cp38-cp38-linux_x86_64.whl` |
```

Comment:
do we have this wheel? cc @will-cromar , would you mind help to confirm this too?

Reply:
We have no torchvision XRT. Deleted for now.